packages/drivers/odsp-driver/src/odspDriverUrlResolver.ts

```diff
@@ -111,7 +111,11 @@ export class OdspDriverUrlResolver implements IUrlResolver {
 			const fileName: string = request.headers[DriverHeader.createNew].fileName;
 			const driveID = searchParams.get("driveId");
 			const filePath = searchParams.get("path");
-			const packageName = searchParams.get("containerPackageName");
+			const packageName =
+				searchParams.get("containerPackageName") ??
+				getContainerPackageName(
```

Comment:
We are not using the headers anymore. So, we can remove this.

Reply:
Added them, please let me know if that's what you were thinking or if you had another idea of how to do it.